user_sync/connector/ldap3_extended/Connection.py

```diff
@@ -0,0 +1,185 @@
+import ldap3
```

Comment:
Don't forget the copyright block at the top of the file.

Reply:
@adorton-adobe Just added the legal stuff to the top